packages/vue/src/components/__tests__/confirm-sign-in.spec.ts

```diff
@@ -64,11 +61,19 @@ describe('ConfirmSignIn', () => {
   });
 
   it('renders as expected for TOTP challenge', () => {
+    // mock random value so that snapshots are consistent
+    const mathRandomSpy = jest.spyOn(Math, 'random').mockReturnValue(0.1);
```

Comment:
I've found that having `Math.random` at the root of spec file causes unexpected side effects on other tests, because it would make all ids the same and mess up accessibility. Moved them inside to snapshot tests to prevent this in the future.